ui/static/ui/js/learning_resources.jsx

```diff
@@ -6,73 +6,179 @@ define('learning_resources', [
   var StatusBox = Utils.StatusBox;
   var Select2 = Utils.Select2;
 
-  var VocabularyOption = React.createClass({
+  var TermList = React.createClass({
     render: function () {
-      var options = _.map(this.props.terms, function (term) {
+      var appliedVocabularies = this.props.vocabs.map(function (vocab) {
+        var selectedTerms = vocab.selectedTerms;
+        var vocabularyName = vocab.vocabulary.name;
+
+        if (selectedTerms.length) {
+          return (
+            <TermListItem
+              key={vocab.vocabulary.id}
+              label={vocabularyName}
+              terms={selectedTerms.join(", ")}
+            />
+          );
+        }
+      }, this);
```

Comment:
It doesn't seem like this argument is necessary here since there is no `this` within the closure